pages/comparison-tool.vue

```diff
@@ -0,0 +1,260 @@
+<template>
+  <div>
+    <ComparisonToolHero />
+    <div class="container">
+      <div class="comparison_card-container mx-auto my-5">
+        <div
+          v-for="(selectedFB, index) in selectedFormBuildersOption"
+          :key="selectedFB + index"
+        >
+          <ComparisonOptionSelectCard
+            :options="filteredOptions(index)"
+            :selected-option="selectedFB"
+            :card-number="index"
+            :form-builders-logo-src="formBuildersLogoSrc"
+            @change="handleOptionChange"
+          />
+        </div>
+      </div>
+      <FormBuilderComparisonDetails
```

Comment:
```suggestion
      <FormBuilderComparisonTable
```

Reply:
Also thinking about the interface. I think it should not be the responsibility of parent component to manage the current selected plan. As we are not doing anything with that. The table itself can handle it and emit the event if required. Looking at a simpler interface like so:

`<FormBuilderComparisonTable :comparing="['formester', 'typeform']" />`

now this component is smart enough to fetch the data for formester and typeform from the right API and reader the information in right format.

Reply:
yeah, right @ankitsinghaniyaz. Selected plans could be moved to the child component. It's not required in the parent. 
And data fetching needs to be present the parent because we are getting the list of available forms from the api, which is used for ComparisonOptionsCard.